### ISSUE TITLE
Image Resize

### ISSUE THREAD

**Issue by jhawlc** [11103212/11103212.github.io#5]
Your images should resize if the pages resize
There should be alt text on each image to describe it.

Example:

`<img src ="images/arms.jpg">`

Should be

`<img src ="images/arms.jpg" alt="Wrist Tech" width=20% height=10%> `
